web-client/src/views/CaseListPractitioner.jsx

```diff
@@ -236,6 +236,65 @@ export const CaseListPractitioner = connect(
                   openTab,
                 )}
             </div>
+            <div className="grid-row display-block">
+              {dashboardExternalHelper.showCaseSearch && <CaseSearchBox />}
+              <div className="card">
+                <div className="content-wrapper gray">
+                  <h3>Filing Fee Options</h3>
+                  <hr />
+                  <p>
+                    <strong>Pay by debit/credit card</strong>
+                    <br />
+                    Copy your docket number(s) and pay online.
+                    <br />
+                    <Button
+                      className="margin-bottom-3 margin-top-2"
+                      href="https://pay.gov/public/form/start/60485840"
+                      id="pay_filing_fee"
+                      target="_blank"
+                    >
+                      Pay now
+                    </Button>
+                  </p>
+                  <hr />
+
+                  <Accordion gray headingLevel="3">
+                    <AccordionItem
+                      customClassName="payment-options"
+                      key={'other-options accordion-icon'}
+                      title={'Other options'}
+                    >
+                      <hr />
+                      <strong>Mail-in payment</strong>
+                      <br />
+                      Make checks/money orders payable to:
+                      <br />
+                      Clerk, United States Tax Court
+                      <br />
+                      400 Second Street, NW
```

Comment:
I don't think a comma should be after Street here 🤔

Reply:
This is indeed how the court currently has the address formatted at the very bottom footer found at https://www.ustaxcourt.gov/

Reply:
Checking with you, @JessicaMarine, should this be changed? (I think it's not typical USPS-recognized formatting)

Reply:
This is fine to leave as-is.  That's how I've always written the address.  I don't think it matters much because the Court's has its own zip code.

Reply:
Whoa, I didn't know that! Thanks 👍